optimum/commands/export/onnx.py

```diff
@@ -54,6 +54,11 @@ def parse_args_onnx(parser):
         default="cpu",
         help='The device to use to do the export. Defaults to "cpu".',
     )
+    optional_group.add_argument(
+        "--fp16",
+        action="store_true",
+        help="Experimental option: use half precision during the export. PyTorch-only, requires `--device cuda`.",
```

Comment:
I'd say experimental because I haven't thouroughly tested it with ONNX Runtime + CUDAExecutionProvider / TensorrtExecutionProvider, and neither with native TensorRT (though in the validation itself we call InferenceSession on CUDA EP, so it's a good sign it's fine). But the export itself is thoroughly tested.